video/types.h

```diff
@@ -299,3 +299,19 @@ uint32_t convertFloatToValue(float rawValue, bool is16Bit, bool isFixed, int8_t
 	}
 };
 
+// This returns an INT32 because it will return either the value
+// in the range 0-65535 or -1 if the string is invalid.
+int32_t textToInt32(const char * text) {
```

Comment:
for consistency with the VDP processor's comms functions, I'd probably be inclined to name this function `textToWord`

the comms function to read a 16-bit value is `readWord_t` (where `_t` just means it can timeout), and it returns a 32-bit value, containing a 16-bit (positive-only) integer.  (there's also a `read24_t` which returns a 32-bit value containing a 24-bit value)